modules/file-integrity-important-attributes.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * security/file_integrity_operator/file-integrity-operator-configuring.adoc
+
+[id="important-file-integrity-object-attributes_{context}"]
+= Important attributes
+
+.Important `spec` and `spec.config` attributes
+
+[%header,cols=2*]
+|===
+|Attribute
+|Description
+
+|`spec.nodeSelector`
+|A map of key-values pairs that must match with node’s labels in order for the
+AIDE pods to be schedulable on that node. The typical use is to set only a
+single key-value pair where `node-role.kubernetes.io/worker: ""` schedules AIDE on
+all worker nodes, `node.openshift.io/os_id: "rhcos"` schedules on all
+{op-system-first} nodes.
+
+|`spec.debug`
+|A boolean attribute. If set to `true`, the daemon running in the AIDE deamon set’s
+pods would output extra information.
+
+|`spec.tolerations`
+|Specify tolerations to schedule on nodes with custom taints. When not specified,
+a default toleration is applied, which allows tolerations to run on master nodes.
+
+|`spec.config.gracePeriod`
+|The number of seconds to pause in between AIDE integrity checks. Frequent AIDE
+checks on a node can be resource intensive, so it can be useful to specify a
+longer interval. Defaults to `3600`, or one hour.
```

Comment:
The default is actually 900, which is 15 mins. Seems like a bug in our upstream docs, I'll fix it up.